docker/otel-collector/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ## base #############################################################################################
-FROM otel/opentelemetry-collector-contrib:0.117.0 AS base
+FROM otel/opentelemetry-collector-contrib:0.120.0 AS base
```

Comment:
we have a few more places to bump this (compose dev + local dockerfile)

Reply:
Did a search on `0.117` and updated the two hits to `0.120`